scripts/directives/cypress-visit-mount-test-example.js

```diff
@@ -8,7 +8,7 @@ const replacer = (text) => {
 }}
 
```

Comment:
Can't for the life of me figure out what's happening here with linting 🤯
https://jmp.sh/l38fDwL